test/test_python_dispatch.py

```diff
@@ -69,6 +69,26 @@ def tearDown(self):
         if hasattr(torch.ops, self.test_ns):
             del torch.ops._test_python_registration
 
+    def test_fallback(self) -> None:
+        with _scoped_library("_", "IMPL") as my_lib:
+            orig_args = ((2, 2),)
+            orig_kwargs = {"device": torch.device("fpga"), "dtype": torch.float64}
+            def my_fallback(op, *args, **kwargs):
+                self.assertIs(op, torch.ops.aten.empty.memory_format)
+                self.assertEqual(args, orig_args)
+                # Why pin_memory was added?
+                self.assertEqual(kwargs, orig_kwargs | {"pin_memory": False})
```

Comment:
To be consistent with torch_dispatch we should be removing default args/kwargs (unless this here is a pre-existing bug). Also I can be convinced to go the other way, I never really liked this convention.

Reply:
I am using the same function torch_dispatch uses to populate these.
It came to my great surprised that pin_memory was added to the kwargs here (and not the other args??)

Reply:
Tracking it in https://github.com/pytorch/pytorch/issues/131772